n/node-csv/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+   "maintainer": "stutiibm",
+   "package_name": "node-csv",
+   "github_url": "https://github.com/adaltas/node-csv",
+   "version": "master",
```

Comment:
@stutiibm this should be same as version you are adding build script

Reply:
Modified build_info.json